lib/ffi-gobject/param_spec.rb

```diff
@@ -12,5 +12,28 @@ def ref
     def accessor_name
       get_name.tr('-', '_')
     end
+
+    def pointer_type?
+      case value_type
+      when TYPE_INT
+        false
+      when TYPE_STRING
+        true
+      end
+    end
+
+    G_TYPE_MAP = {
+      TYPE_INT => :gint,
+      TYPE_STRING => :utf8,
+      TYPE_LONG => :glong
+    }.freeze
```

Comment:
The map seems to fail for unhandled types. I've got a GObject property and a non-fundamental (but GBoxed-derived) types which don't work with it 
```
/home/kugel/.gem/ruby/2.3.0/gems/gir_ffi-0.10.2/lib/ffi-gobject/param_spec.rb:35:in `fetch': key not found: 19016480 (KeyError)
	from /home/kugel/.gem/ruby/2.3.0/gems/gir_ffi-0.10.2/lib/ffi-gobject/param_spec.rb:35:in `type_tag'
	from /home/kugel/.gem/ruby/2.3.0/gems/gir_ffi-0.10.2/lib/ffi-gobject/param_spec.rb:39:in `ffi_type'
```